WebRoot/css/zm.css

```diff
@@ -7679,3 +7669,91 @@ input[type="radio"] + label {
 .ZmAppointmentTooltip TD.ZmAppointmentTooltipValue {
 	@AppointmentTooltipValue@	
 }
+
+/* Appointment create/edit view */
+.ZmApptEditView {
+	@ApptEditView@
+}
+
+.ZmApptEditView * {
+	@ApptEditViewFontSize@
```

Comment:
Can we rename this to ApptEditViewElements? so in future we can use same property to apply more styling if required

Reply:
Fixed: https://github.com/Zimbra/zm-web-client/pull/143/commits/b15e5e1cd57ec3595682a88031056b2ef693148d#diff-33f9c57f22cd743824b2918132c5d73bR7679 and https://github.com/Zimbra/zm-web-client/pull/143/commits/b15e5e1cd57ec3595682a88031056b2ef693148d#diff-16ae02866e4f63813de9a7825a1057e3R2563

Thanks